packages/lib/gpcircuits/test/proto-pod-gpc.spec.ts

```diff
@@ -393,12 +420,32 @@ function makeTestSignals(
     sigEntryProofDepth.push(BigInt(entrySignals.proof.siblings.length));
     sigEntryProofIndex.push(BigInt(entrySignals.proof.index));
 
-    // Fillin sibling array, padded with 0s to max length.
+    // Fill in sibling array, padded with 0s to max length.
     sigEntryProofSiblings.push(
       extendedSignalArray(entrySignals.proof.siblings, paramMaxDepth)
     );
   }
 
+
+  // ?
+  const indexListPairs = [
+    [1, [85n, sigEntryValue[1]].map(x => { return { type: "cryptographic", value: x }; })],
```

Comment:
This use of .map() for 2-3 constant entries seems questionable to me.  Harder to read than the literals, and not much shorter.